build.gradle

```diff
@@ -3,9 +3,7 @@ buildscript {
     repositories {
         // Make sure that you have the following two repositories
         google()  // Google's Maven repository
-
         mavenCentral()  // Maven Central repository
-        jcenter()  // jcenter repository
```

Comment:
why removing this one ?

Reply:
I'm removing this line because:

- jcenter is deprecated
- we don't need it right now (and never will)